src/core_plugins/console/api_server/es_6_0/query.js

```diff
@@ -625,6 +664,10 @@ export default function (api) {
         __one_of: ['avg', 'total', 'max', 'none'],
       },
     },
+    percolate: {
+      field: '',
+      document: {}
+    },
```

Comment:
I think more parameters need to be supported? https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-percolate-query.html#_parameters_10